testsuite/fmi-cross-check/importFMUs.py

```diff
@@ -28,11 +28,12 @@ def generateLua(modelName, testFMUDir, resultDir, fmiType):
   luaFilePath = os.path.join(resultDir, modelName + ".lua")
 
   # Set OMSimulator settings
-  tempDir = "temp"
+  tempDir = "/temp/cross-check/"
```

Comment:
@AnHeuermann This might solve the issue with too long file names, right? Maybe we have to delete the temp folders in this path from time to time... any suggestions?

Reply:
This probably won't work on Windows, but should be fine for some Unix systems. We could just use some Python function to get the temp files directory, e.g the way described in https://stackoverflow.com/questions/847850/cross-platform-way-of-getting-temp-directory-in-python.

Reply:
Good point 👍